packages/big-design/src/components/Button/spec.tsx

```diff
@@ -188,24 +182,51 @@ test('render only icon only with left and right icons button', () => {
   expect(screen.getAllByTestId('icon-only')).toHaveLength(1);
 });
 
-test('hides content when isLoading is true', () => {
-  const plusIcon = <AddIcon role="icon" aria-hidden="false" />;
-  const { rerender } = render(
-    <Button isLoading={false} iconLeft={plusIcon}>
-      Button
-    </Button>,
-  );
-  const visibleContent = screen.getByRole('icon');
+describe('isLoading', () => {
```

Comment:
Create a section for `isLoading` since we seem to have several tests validating this state.